backend/typescript/graphql/resolvers/adminResolvers.ts

```diff
@@ -0,0 +1,43 @@
+import AdminService from "../../services/implementations/adminService";
+import {
+  IAdminService,
+  NotificationDTO,
+} from "../../services/interfaces/adminService";
+
+// change staff_id to StaffService when its done
```

Comment:
Can you prepend a `TODO:` to this comment? It'll make it easier to Find Everywhere after